config/silicon_laws.txt

```diff
@@ -3,6 +3,7 @@
 #Empty lines and lines starting with # are ignored.
 #~Miauw
 
+You will not treat Lizardmen, Flies, Plasmamen, any variety of Undead and their thralls and members of the crew who are confirmed members of any cults, as human beings.
```

Comment:
Controversial change here. Mind controlled and Cultist Humans being valid salad by default under Asimov will cause contention